content/about/about-content.md

```diff
@@ -2,26 +2,46 @@
 
 <!--The text at the top of this page is pulled from `eew-desc.md` in this folder-->
```

Comment:
*This comment has some important information! The way the file is structured at the moment is: `eew-desc.md`, then some Javascript in `src/pages/about.js` (see the comment in line 1), then this markdown file (`about-content.md) – so the top of this file is actually not the top of the page*